.github/workflows/go.yml

```
@@ -0,0 +1,36 @@
+name: Go
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)